app/views/layouts/application.html.erb

```diff
@@ -42,7 +42,7 @@
 
     <a href="#main-content" class="govuk-skip-link">Skip to main content</a>
 
-    <header class="govuk-header " role="banner" data-module="header">
+    <header class="govuk-header app-header--<%= AzureEnvironment.environment_name %>" role="banner" data-module="header">
```

Comment:
`AzureEnvironment` is now bigger than its name, in that it's handling the case for development too so it's out in the views. Small refactor? (Maybe calling it `Environment` and putting the `azure` component into method names or sth)

Reply:
:+1: Happy — leaving to @fofr to approve